controllers/semeru_compiler.go

```diff
@@ -302,6 +302,23 @@ func (r *ReconcileWebSphereLiberty) reconcileSemeruDeployment(wlva *wlv1.WebSphe
 			Annotations: wlutils.GetWLOLicenseAnnotations(),
 		},
 		Spec: corev1.PodSpec{
+			Affinity: &corev1.Affinity{
```

Comment:
We should consider the following:
- add anti-affinity for host topology
- add affinity so that semeru pods are scheduled on the same nodes as Liberty application pods